SWLOR.Game.Server/Data/Migrations/2018-12-10.2.sql

```diff
@@ -0,0 +1,6 @@
+﻿  UPDATE dbo.SpawnObject
+  SET BehaviourScript = 'PackAggroBehaviour'
+  WHERE NPCGroupID IS NOT NULL
```

Comment:
This will modify the behaviours for all NPCs. I don't know that this is correct. There are several different AI groups in the DB: SightAggroBehaviour, SoundAggroRandomWalkBehaviour, SoundAggroBehaviour

You follow up with a fix to the SightAggroBehaviour ones, but the SoundAggroRandomWalkBehaviour entries are still going to get updated.

Reply:
I'll go through it and edit the SoundAggro entries back to default

Reply:
Resolved it by just making it change sightaggros to pack